### ISSUE TITLE
Improving ANN to outperform  RandomForestClassifier

### ISSUE THREAD

**Issue by marcinpawelkot** [fchollet/keras#6641]
Hello,

I am trying to build a binary classifier for flight delays. I have 16 explaining variables, training data set of 10k records and test data set of 50k records. Currently, my classifier built with Keras has an accuracy of 75%, while RandomForestClassifier from scikitlearn on the same data has a little above 77%. My aim is to outperform RandomForest, however, i am stuck - i 've tried different scalers, sizes of data sets, structures of neural network and its properties. Here is structure of me classifier:

model = Sequential()
model.add(Dense(500, activation='relu', input_dim=16))
model.add(Dropout(0.5))
model.add(Dense(750, activation='relu'))
model.add(Dropout(0.5))
model.add(Dense(2, activation='softmax'))
sgd = SGD(lr=0.01, decay=1e-6, momentum=0.9, nesterov=True)
model.compile(loss='categorical_crossentropy',
                  optimizer=sgd,
                  metrics=['accuracy'])

I will be grateful for any advice!

**Comment by kgrm** [comment]
Consider using different optimizers (eg, RMSprop, Adam, NAdam), weight regularization, batch normalization, etc.

**Comment by yinjianhong** [comment]
bn
